### ISSUE TITLE
Support for WDS_SOCKET_PORT=0

### ISSUE THREAD

**Issue by markNZed** [Sean-Bradley/R3F-Pack#7]
I'm trying to use GitHub Codespace with r3f and Github maps port 3000 to a URL so I needed to set WDS_SOCKET_PORT=0 when using react-scripts and it seems rsf-pack does not support this.

**Comment by Sean-Bradley** [comment]
I have updated to support WDS_SOCKET_PORT. 

You will need to test it since I don't really know if it will work like this in CodeSpaces.

Update r3f-pack to latest (2.8.1)
```
npm install r3f-pack@latest
```

In your `package.json`, update the `start` comand to pass in `WDS_SOCKET_PORT`

In Windows,
```
"scripts": {
    "start": "set WDS_SOCKET_PORT=0 && r3f-pack start",
```
In a Unix shell
```
"scripts": {
    "start": "export WDS_SOCKET_PORT=0 && r3f-pack start",
```

**Comment by markNZed** [comment]
Thanks very much! I had some pain uninstalling react-scripts and putting them back so I will have a go at this once I have pushed what I'm working on now.

**Comment by Sean-Bradley** [comment]
It works.

I tested by creating a CodeSpace from my boilerplates `OrbitControls` branch.

[https://github.com/Sean-Bradley/React-Three-Fiber-Boilerplate/tree/orbitControls](https://github.com/Sean-Bradley/React-Three-Fiber-Boilerplate/tree/orbitControls)

In the CodeSpace version, I edited the `start` command in the `package.json`

```
"start": "export WDS_SOCKET_PORT=0 && r3f-pack start",
```

Then from the CodeSpace terminal, I ran,
```
npm start
```

and it worked as expected with HMR live reload when editing components.
